week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -10,10 +10,31 @@
   Use the tests to confirm which message to return
 */
 
-function bushChecker() {
-
+function bushChecker(listOfBerryColours) {
+  for(let i=0; i < listOfBerryColours.length; i++){
```

Comment:
This works, but how could we do it with an array method? It looks like you were really close in the commented out code below!

Reply:
The commented code was supposed to be my final version but committed the task after trying out other methods. I have committed the  intended version.